src/include/rbd/librbd.hpp

```diff
@@ -38,6 +39,7 @@ namespace librbd {
     uint64_t id;
     uint64_t size;
     std::string name;
+    utime_t stamp;
```

Comment:
Right... so could you give me some suggestion? I still believe the timestamp of a snapshot is very import to be listed.